From a4e353cbfee7b2c593bd692c3a439b1e41797678 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan.agner@toradex.com>
Date: Tue, 27 Mar 2018 18:55:36 +0200
Subject: [PATCH 1/2] remove DPI handling

It looks ugly on high res screen and Toradex Easy Installer does
not need it since anyway only VGA is used.

Signed-off-by: Stefan Agner <stefan.agner@toradex.com>
---
 src/gui/embedded/qscreenlinuxfb_qws.cpp | 12 +++---------
 1 file changed, 3 insertions(+), 9 deletions(-)

diff --git a/src/gui/embedded/qscreenlinuxfb_qws.cpp b/src/gui/embedded/qscreenlinuxfb_qws.cpp
index a86687d8e4..521b07c0b1 100644
--- a/src/gui/embedded/qscreenlinuxfb_qws.cpp
+++ b/src/gui/embedded/qscreenlinuxfb_qws.cpp
@@ -426,15 +426,9 @@ bool QLinuxFbScreen::connect(const QString &displaySpec)
             physWidth = dw*physHeight/dh;
     }
     if (dimIdxW < 0 && dimIdxH < 0) {
-        if (vinfo.width != 0 && vinfo.height != 0
-            && vinfo.width != UINT_MAX && vinfo.height != UINT_MAX) {
-            physWidth = vinfo.width;
-            physHeight = vinfo.height;
-        } else {
-            const int dpi = 72;
-            physWidth = qRound(dw * 25.4 / dpi);
-            physHeight = qRound(dh * 25.4 / dpi);
-        }
+        const int dpi = 72;
+        physWidth = qRound(dw * 25.4 / dpi);
+        physHeight = qRound(dh * 25.4 / dpi);
     }
 
     dataoffset = yoff * lstep + xoff * d / 8;
-- 
2.16.2

